lib/converters/circuit-to-3d.ts

```diff
@@ -38,7 +48,80 @@ import type { Vec2 } from "@jscad/modeling/src/maths/types"
 
 const DEFAULT_BOARD_THICKNESS = 1.6 // mm
 const DEFAULT_COMPONENT_HEIGHT = 2 // mm
+const DEFAULT_BOARD_DIMENSION = 100 // mm
```

Comment:
never define a constant twice, and there's no reason for default board sizes as i mentioned above